<launch>
    <!-- Autogenerated launch file - do not hand edit -->
    <include file="/home/albert/ros/mct_config/machine/mct.machine" />
    
    <group ns="/mct_master/camera_11/camera">
        <node pkg="mct_frame_skipper" type="frame_skipper.py" name="frame_skipper" args="/mct_master/camera_11/camera/image_rect 3" machine="mct_master" />
    </group>
    
    <group ns="/mct_master/camera_3/camera">
        <node pkg="mct_frame_skipper" type="frame_skipper.py" name="frame_skipper" args="/mct_master/camera_3/camera/image_rect 3" machine="mct_master" />
    </group>
    
    <group ns="/mct_master/camera_6/camera">
        <node pkg="mct_frame_skipper" type="frame_skipper.py" name="frame_skipper" args="/mct_master/camera_6/camera/image_raw 3" machine="mct_master" />
    </group>
    
    <group ns="/mct_master/camera_8/camera">
        <node pkg="mct_frame_skipper" type="frame_skipper.py" name="frame_skipper" args="/mct_master/camera_8/camera/image_rect 3" machine="mct_master" />
    </group>
    
    <group ns="/mct_slave1/camera_1/camera">
        <node pkg="mct_frame_skipper" type="frame_skipper.py" name="frame_skipper" args="/mct_slave1/camera_1/camera/image_rect 3" machine="mct_slave1" />
    </group>
    
    <group ns="/mct_slave1/camera_2/camera">
        <node pkg="mct_frame_skipper" type="frame_skipper.py" name="frame_skipper" args="/mct_slave1/camera_2/camera/image_rect 3" machine="mct_slave1" />
    </group>
    
    <group ns="/mct_slave1/camera_4/camera">
        <node pkg="mct_frame_skipper" type="frame_skipper.py" name="frame_skipper" args="/mct_slave1/camera_4/camera/image_rect 3" machine="mct_slave1" />
    </group>
    
    <group ns="/mct_slave1/camera_9/camera">
        <node pkg="mct_frame_skipper" type="frame_skipper.py" name="frame_skipper" args="/mct_slave1/camera_9/camera/image_rect 3" machine="mct_slave1" />
    </group>
    
    <group ns="/mct_slave2/camera_10/camera">
        <node pkg="mct_frame_skipper" type="frame_skipper.py" name="frame_skipper" args="/mct_slave2/camera_10/camera/image_rect 3" machine="mct_slave2" />
    </group>
    
    <group ns="/mct_slave2/camera_12/camera">
        <node pkg="mct_frame_skipper" type="frame_skipper.py" name="frame_skipper" args="/mct_slave2/camera_12/camera/image_rect 3" machine="mct_slave2" />
    </group>
    
    <group ns="/mct_slave2/camera_5/camera">
        <node pkg="mct_frame_skipper" type="frame_skipper.py" name="frame_skipper" args="/mct_slave2/camera_5/camera/image_rect 3" machine="mct_slave2" />
    </group>
    
    <group ns="/mct_slave2/camera_7/camera">
        <node pkg="mct_frame_skipper" type="frame_skipper.py" name="frame_skipper" args="/mct_slave2/camera_7/camera/image_rect 3" machine="mct_slave2" />
    </group>
     
</launch>